### PR TITLE
Use ES2016 snapshot links

### PR DESCRIPTION
**Pull request by chicoxyzzy**
As per discussion from #828

Please check misc links in this PR because I'm not 100% all of them are ok.

cc @bterlson

### CODE REVIEW

File: data-es2016plus.js [hunk 1]
```diff
@@ -440,7 +440,7 @@ exports.tests = [
     name: 'generator functions can\'t be used with "new"',
     category: '2016 misc',
     significance: 'tiny',
-    link: 'https://github.com/rwaldron/tc39-notes/blob/master/es7/2015-07/july-28.md#67-new--generatorfunction',
+    link: 'http://www.ecma-international.org/ecma-262/7.0/index.html#sec-createdynamicfunction',
```
Comment (ljharb) [inline data-es2016plus.js]: This link seems useful to keep somewhere

File: data-es2016plus.js [hunk 2]
```diff
@@ -464,7 +464,7 @@ exports.tests = [
     name: 'generator throw() caught by inner generator',
     category: '2016 misc',
     significance: 'tiny',
-    link: 'https://github.com/tc39/ecma262/issues/293',
+    link: 'http://www.ecma-international.org/ecma-262/7.0/index.html#sec-generatorfunction-objects',
```
Comment (ljharb) [inline data-es2016plus.js]: Also this

File: data-es2016plus.js [hunk 3]
```diff
@@ -496,7 +496,7 @@ exports.tests = [
     name: 'strict fn w/ non-strict non-simple params is error',
     category: '2016 misc',
     significance: 'tiny',
-    link: 'https://github.com/rwaldron/tc39-notes/blob/master/es7/2015-07/july-29.md#611-the-scope-of-use-strict-with-respect-to-destructuring-in-parameter-lists',
+    link: 'http://www.ecma-international.org/ecma-262/7.0/index.html#sec-functiondeclarationinstantiation',
```
Comment (ljharb) [inline data-es2016plus.js]: Etc

**Review by ljharb — CHANGES_REQUESTED** [review]
I like using snapshots, but we'd lose a lot of important information by erasing the links to meeting notes and github issues. Not sure what the best solution is.

**Comment by chicoxyzzy** [comment]
@ljharb should I add another field for old links or just add a comment or revert to old links?

**Comment by ljharb** [comment]
I think all of those are good options - maybe we should change "link" to be an object of "link name" to "link url", and then we can display them all in the UI?

**Comment by bterlson** [comment]
There are two concepts here: links to spec text, and links to supporting info. Spec text should probably have a "spec:" key, other links can go into a links array and be displayed as superscript footnotes or something?

**Comment by chicoxyzzy** [comment]
Updated

**Comment by zloirock** [comment]
LGTM

**Comment by zloirock** [comment]
However, I think it would be better to make notes better readable, not just links with 2 words.

**Comment by zloirock** [comment]
"Meeting notes" - from what meeting? "GitHub issue" - which issue from which repo? Etc.

**Comment by chicoxyzzy** [comment]
OK I'll try to add more meaningful notes

**Comment by chicoxyzzy** [comment]
Please let me know if I can improve footnotes even more

**Comment by zloirock** [comment]
LGTM

**Comment by zloirock** [comment]
Inconsistency

```
[9] TC39 meeting notes from 28 July, 2015.
[10] TC39 meeting notes from July 28, 2015.
```

**Comment by chicoxyzzy** [comment]
thank you. fixed

**Comment by chicoxyzzy** [comment]
I'm still not sure that current spelling in footnotes is good enough. @ljharb @bterlson can you re-review please?

**Review by ljharb — APPROVED** [review]

**Pull request merged**